builtin/game/privileges.lua

```diff
@@ -90,3 +90,7 @@ core.register_privilege("debug", {
 	description = "Allows enabling various debug options that may affect gameplay",
 	give_to_singleplayer = false,
 })
+core.register_privilege("clearinv", {
+	description = "Can use clear inventory command",
```

Comment:
Maybe just require `server` priv... IDK. For now, I'll change desc and leave priv name until better suggestion.